activities/tool-management/index.md

```diff
@@ -24,6 +24,7 @@ Our current tools are:
 * [Jitsi Meet](jitsi-meet.md) (videoconferencing)
 * [Mattermost](mattermost.md) (chat)
 * [Odoo Community](odoo.md) (CRM and project management)
+* [Nextcloud](nextcloud.md) (file sharing)
```

Comment:
If we want it in alphabetical order, this should move up one step.

Reply:
I never consciously put it in alphabetical order, interesting! But sure, I can do that, it makes sense and will appease neurotic people as well :)